linux_os/guide/system/logging/log_rotation/ensure_logrotate_activated.rule

```diff
@@ -1,7 +1,5 @@
 documentation_complete: true
 
-prodtype: rhel6,rhel7,fedora
```

Comment:
Apparently it runs in OpenSUSE and people have lots of questions about it: https://forums.opensuse.org/showthread.php/505897-On-a-fresh-installation-is-logrotate-running-by-default

Reply:
Also Ubuntu: https://manpages.debian.org/jessie/logrotate/logrotate.8.en.html